### PR TITLE
Ensure 'grr preview' processes all previewable resources

### LINKED ISSUE #139: "grr preview" sometimes doesn't generate snapshots for all dashboards
I see nondeterministic behaviour with `grr preview` where sometimes it only generates snapshots for a random subset of the dashboards defined in the input.

Rerunning the command will result in a different set, sometimes I have to run it upwards of 5 times before it generates a snapshot for the dashboard I wish to actually preview.

### PR DESCRIPTION
**Pull request by rhowe**
Instead of stopping when we encounter a resource which doesn't support previewing (e.g. a DashboardFolder), just skip it with a warning.

Fixes #139

### CODE REVIEW

**Comment by rhowe** [comment]
@dsotirakis not wanting to bug you, but what do you think of this?

Also we have been running grizzly with https://github.com/grafana/grizzly/pull/140 applied and find it more usable that way

**Review by dsotirakis — APPROVED** [review]
I think it does make sense - sorry for missing that @rhowe, LGTM!

Ready to merge if @malcolmholmes is also happy with it :)

**Review by malcolmholmes — APPROVED** [review]
This makes sense, thanks.

**Pull request merged**
